src/main/java/com/ceritoxi/util/Math.java

```diff
@@ -1,130 +1,457 @@
 package com.ceritoxi.util;
+import java.util.Stack;
 
+public class Meth {
+
+    public static boolean isDividableByXorY(int dividend, int x, int y) {
+        if ((x != 0 && dividend % x == 0) || (y != 0 && dividend % y == 0)) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfNumbersUnderZThatAreDividableByXorY (int x, int y, int z) {
+        int sum = 0;
+
+        if (z < 1) {
+            return sum;
+        }
+        for (int i = 1; i < z; i++) {
+            if (isDividableByXorY(i, x, y)) {
+                sum += i;
+            }
+        }
+        return sum;
+    }
+
+    public static boolean isEven (int number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static boolean isEven (long number) {
+        if (number % 2 == 0) {
+            return true;
+        }
+        return false;
+    }
+
+    public static int sumOfEvenFibonacciTermsUnderX (int x) {
```

Comment:
But where is the consistency in it then? I named it underX so it should be x imo. If I name it upperBound you would need an extra step of thinking to understand what that upperBound stands for. Yes it may be more logical in a sense, but if I would rename the argument I would rename the whole function as well.

Reply:
you have to rename the function sure. However you just have to truncate the name since the paramater name will tell the rest such as: 
```public static int sumOfEvenFibonacciTerms (int upperBound) {...}``` 
From that its obvious that we do not want to count the sum to infinity and the upperBound tells the rest.